src/cmds/qsub_sup.c

```diff
@@ -207,6 +207,37 @@ get_conf_path(void)
 	}
 }
 
+/**
+ * @brief
+ *      Check the line does not end with win/mac.
+ *
+ * @param[in]	s	- input line
+ *
+ * @return      int
+ * @retval -1 - input error
+ * @retval 0 - unix (not win/mac)
+ * @retval 1 - mac (line ends with '\r')
```

Comment:
Are you sure about this?  MacOS is unix like.  I don't have \r issues on my mac.

Reply:
Well, from my testing I am quite sure. unix2mac converts to `\r` and the script can not run.
```
torque1.grid.cesnet.cz$ cat job.sh
#!/bin/bash

lsb_release -a
torque1.grid.cesnet.cz$ ./job.sh 
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 12 (bookworm)
Release:	12
Codename:	bookworm
torque1.grid.cesnet.cz$ unix2mac job.sh
unix2mac: converting file job.sh to Mac format...
torque1.grid.cesnet.cz$ ./job.sh 
-bash: ./job.sh: cannot execute: required file not found
```

Also tested as job:
```
torque1.grid.cesnet.cz$ qsub job.sh
94018.torque1.grid.cesnet.cz
torque1.grid.cesnet.cz$ cat job.sh.e94018
-bash: line 1: /var/spool/pbs/mom_priv/jobs/94018.torque1.grid.cesnet.cz.SC: cannot execute: required file not found
```

Reply:
I realized I used bash. Is `bash` the shell on your cluster? Maybe other shells are able to run Mac files on Linux??? Although, I am not sure...

Reply:
Oh, I found the answer. I can see now it is an old matter. Single `\r` is on old Macs OS 9 and before - the year 1999 and before... I added the check based on the Linux utility unix2mac. Should I remove the Mac check? It can not do any harm but it will probably not engage.

Reply:
I'd remove the mac part.  MacOS 9 is like 30+ years old.  It was back before MacOS became UNIX like.

Reply:
Thanks @bhroam. Done.